actions/remove.go

```diff
@@ -22,16 +22,17 @@ import (
 //RemoveCommand to remove all codewind and project images
 func RemoveCommand(c *cli.Context) {
 	tag := c.String("tag")
-	imageArr := [4]string{}
+	imageArr := [5]string{}
 	imageArr[0] = "eclipse/codewind-pfe"
 	imageArr[1] = "eclipse/codewind-performance"
 	imageArr[2] = "eclipse/codewind-initialize"
 	imageArr[3] = "cw-"
+	imageArr[4] = "appsody"
```

Comment:
Is this removing appsody images?

Reply:
It's not removing them, but wasn't supposed to be left in the list. https://github.com/eclipse/codewind-installer/pull/106 removes it.